src/quo/components/text_input.cljs

```diff
@@ -66,9 +66,9 @@
       false)))
 
 ;; TODO(Ferossgp): Check performance for android layout animations
-;; (when (and platform/android?
-;;            (aget rn/ui-manager "setLayoutAnimationEnabledExperimental"))
-;;   (ocall rn/ui-manager "setLayoutAnimationEnabledExperimental" true))
+(when (and platform/android?
```

Comment:
hows this related to this PR ?

Reply:
Here I used Layout animation to animate the cancel button on iOs, to use the same animation on android, it should be enabled like that. I commented that code because at some point I thought that we will never show cancel on android (forgot about the search input)